packages/client/src/client4.ts

```diff
@@ -747,10 +746,9 @@ export default class Client4 {
         return profile;
     };
 
-    loginById = (id: string, password: string, token = '', deviceId = '') => {
```

Comment:
Same here

Reply:
i added them back in the body to not break the public API specs